contrib/cirrus/runner.sh

```diff
@@ -233,6 +233,24 @@ function _run_consistency() {
     SUGGESTION="run 'make generate-bindings' and commit all changes" ./hack/tree_status.sh
     make completions
     SUGGESTION="run 'make completions' and commit all changes" ./hack/tree_status.sh
+
+    if  [[ -z "$CIRRUS_TAG" ]] && \
+        req_env_vars CIRRUS_CHANGE_IN_REPO CIRRUS_PR DEST_BRANCH
+    then
+        msg "#####"
+        msg "Verifying no change adds new calls to io/ioutils."
```

Comment:
The package is called `io/ioutil` without s. Please fix this also in the commit message, and the regex below

Reply:
Good catch, thanks.